assets/animations.js

```diff
@@ -13,10 +13,16 @@ function onIntersection(elements, observer) {
   });
 }
 
-function initializeScrollAnimationTrigger(rootEl = document) {
+function initializeScrollAnimationTrigger(rootEl = document, isDesignMode = false) {
   const animationTriggerElements = Array.from(rootEl.getElementsByClassName(SCROLL_ANIMATION_TRIGGER_CLASSNAME));
   if (animationTriggerElements.length === 0) return;
 
+  if (isDesignMode) {
```

Comment:
Is `isDesignMode` simply `Shopify.designMode` or is there another use case where these won't be equal?

If they're the same thing, do we need this another variable/parameter instead of simply checking for `Shopify.designMode`?